### PR TITLE
Change the `thumbp` role to install the Node.js rewrite

### PR DESCRIPTION
**Pull request by markbreedlove**
See https://github.com/dpla/thumbp/pull/4

I'll need to squash this before it's merged. "Fix ES port in init script" (499c732) should probably remain its own commit, but the others should probably be combined.

### CODE REVIEW

File: ansible/roles/thumbp/files/build_thumbp.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+cd /opt/thumbp/src/thumbp || exit_w_error "Can not cd to /opt/thumbp/src/thumbp"
+
+exit_w_error() {
+    echo >&2 $1
+    echo $1 >> $log
+    exit 1
+}
+
+echo "Ensuring correct version of Node" | tee -a $log
+nvm install || exit_w_error "nvm install failed"
```
Comment (moltude) [inline ansible/roles/thumbp/files/build_thumbp.sh]: @markbreedlove when I tested this locally it was failing at this step until I added $HOME/.nvmrc. Can you confirm?   Does it need to create a .nvmrc file as a part of install_nvm.sh?

Comment (markbreedlove) [inline ansible/roles/thumbp/files/build_thumbp.sh]: Can you confirm that you have `thumbp_branch_or_tag: fix/switch-to-node` in your `group_vars/development` file? That should have an `.nvmrc`.

Comment (moltude) [inline ansible/roles/thumbp/files/build_thumbp.sh]: that got it; thanks

**Comment by moltude** [comment]
this looks 👍

**Comment by markbreedlove** [comment]
OK, I've actually squashed it into three commits. I decided that the logrotate commit should be its own thing.

**Pull request merged**